addon/components/frost-list.js

```diff
@@ -0,0 +1,127 @@
+import Ember from 'ember'
+const {Component} = Ember
+import computed from 'ember-computed-decorators'
+import layout from '../templates/frost-list'
+import {PropTypes} from 'ember-prop-types'
+import SlotsMixin from 'ember-block-slots'
+import _ from 'lodash'
+
+const FrostList = Component.extend(SlotsMixin, {
+
+  // == Dependencies ==========================================================
+
+  // == Properties ============================================================
+  classNames: ['frost-list'],
+  layout: layout,
+  records: Ember.computed.alias('model'), // TODO Log deprecation of model as an attribute
```

Comment:
Since we're going to a major here, I actually think `items` is the correct term - e.g. <li>